Kernel/Memory/RegionTree.cpp

```diff
@@ -121,6 +121,23 @@ ErrorOr<VirtualRange> RegionTree::allocate_range_randomized(size_t size, size_t
         if (!m_total_range.contains(random_address, size))
             continue;
 
+#if ARCH(I386)
```

Comment:
Please add a small comment here describing what this is about. :^)

Reply:
Done (?)